packages/xo-server/src/api/disk.js

```diff
@@ -86,7 +89,7 @@ async function handleExportContent (req, res, { xapi, id }) {
   )
   pump(stream, res, error => {
     if (error != null) {
-      console.warn('disk.exportContent', error)
+      log.warn(`disk.exportContent ${error}`)
```

Comment:
Error is probably an `Error` object and not a stream, concatenating is not a good idea here.

Furthermore, `@xen-orchestra/log` accepts a `data` params which may contain an error object:

```js
log.warn('disk.exportContent', { error })
```